### PR TITLE
COS packaging for KSQL.

### PR DESCRIPTION
**Pull request by hjafarpour**
Changes to make KSQL packaging in COS possible.

### CODE REVIEW

File: ksql-package/src/assembly/package.xml [hunk 1]
```diff
@@ -2,34 +2,57 @@
...
+         -->
     <id>package</id>
     <formats>
         <format>dir</format>
-        <format>tgz</format>
-        <format>zip</format>
     </formats>
-    <includeBaseDirectory>true</includeBaseDirectory>
-    <baseDirectory>ksql</baseDirectory>
+    <includeBaseDirectory>false</includeBaseDirectory>
```
Comment (rodesai) [inline ksql-package/src/assembly/package.xml]: why get rid of the base directory in the package tarball?

File: ksql-package/src/assembly/package.xml [hunk 2]
```diff
@@ -2,34 +2,57 @@
...
           xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2
           http://maven.apache.org/xsd/assembly-1.1.2.xsd">
-    <!-- Assembles a package that can run ksql -->
+    <!-- Assembles a package that can run ksql given dependency packages. This is used to
+         construct packages OS packages where underlying libraries, like rest-utils, can be shared
+         -->
     <id>package</id>
     <formats>
         <format>dir</format>
-        <format>tgz</format>
```
Comment (rodesai) [inline ksql-package/src/assembly/package.xml]: We upload this tgz to S3 for releases

**Comment by hjafarpour** [comment]
@rodesai Added the removed lines again.

File: ksql-package/src/assembly/package.xml [hunk 3]
```diff
@@ -2,34 +2,59 @@
...
+            <outputDirectory>share/doc/ksql/</outputDirectory>
             <includes>
+                <include>version.txt</include>
+                <include>README*</include>
                 <include>LICENSE*</include>
+                <include>NOTICE*</include>
                 <include>licenses/</include>
                 <include>notices/</include>
-                <include>bin/</include>
-                <include>config/</include>
```
Comment (rodesai) [inline ksql-package/src/assembly/package.xml]: It should work if you add this back in, and get rid of lines 35-41. Or change etc/ksql on line 37 to config.

**Pull request merged**